l10n_ro_config/views/account_journal.xml

```diff
@@ -31,6 +31,16 @@
                     attrs="{'invisible': [('is_l10n_ro_record','=',False)]}"
                 />
             </field>
+            <xpath expr="//group[@name='group_alias_edit']" position="after">
+                <field name="is_l10n_ro_record" invisible="1" />
+                <group
```

Comment:
Nu ar trebui ca acest grup sa fie invizibil pe jurnalele cash, bank, sale?

Reply:
poate fi si sale...daca e autofactura de la furnizor in numele tau